### ISSUE TITLE
Cannot catch error because of uncaught promise

### ISSUE THREAD

**Issue by Lino-LP** [DethAriel/ng-recaptcha#194]
### Summary
An uncaught promise prevents the use of rxjs Observer's onError catching the error if something goes wrong.

I'm submitting a: 
- [X] bug report
- [ ] feature request
- [ ] question / support request 
- [ ] other

### Description

When an error occurs with the recaptcha service (e.g. incorrect RECAPTCHA_V3_SITE_KEY), an uncaught promise results in an unrecoverable error.
```
Error: Uncaught (in promise): Error: Invalid site key or not loaded in api.js: <incorrect key>
Error: Invalid site key or not loaded in api.js: <incorrect key>
    at Array.<anonymous> (recaptcha__en.js:207)
    at recaptcha__en.js:328
    at ng-recaptcha.js:289
    at ZoneDelegate.invoke (zone-evergreen.js:364)
    at Zone.run (zone-evergreen.js:123)
    at NgZone.runOutsideAngular (core.js:41670)
    at ReCaptchaV3Service.executeActionWithSubject (ng-recaptcha.js:287)
    at ReCaptchaV3Service.execute (ng-recaptcha.js:280)
```

To reproduce the issue, simply use an incorrect RECAPTCHA_V3_SITE_KEY

Then in a controller, use this code
`this.recaptchaV3Service.execute('register').subscribe((token) => this.doSomething(token), (error) => { console.log('this is never reached'); });`

The browser console displays the error message above (Error: Uncaught (in promise)), and the error block is not called.

Lib versions:

* ng-recaptcha: 6.0.2
* Angular: 9.1.0
* Typescript: 3.8.3

**Comment by DethAriel** [comment]
Hi @Lino-LP !

Providing an incorrect site key to the module seems like to be a pretty fatal flaw. Can you please expand on your use-case so that I can understand it better, and prioritize the fix accordingly?

**Comment by Lino-LP** [comment]
Hi @DethAriel 

The incorrect site key was the easiest way I found to reproduce this issue, but we actually found it by not having our domain in the reCaptcha domains whitelist.

As we are dealing with white-labels, this configuration step will happen frequently and might be incorrectly executed every now and then.
It would be much more reassuring to us to make sure the feature doesn't fully crash on users when this happens, and to catch the error and log it into our error logging system so we can catch it and fix it.

**Comment by DethAriel** [comment]
@Lino-LP 

are you saying that the whole Angular app crashes when this happens? Or did you mean just the reCAPTCHA functionality? If it is the latter -- what would be your fallback in that case?

**Comment by Lino-LP** [comment]
@DethAriel 

The reCAPTCHA functionality stops, and as a result users cannot register and are stuck with a loader and no error message.
If I could catch the error, I would remove the loader immediately and let the user know we are having issues and we'll fix it ASAP, and collect their expression of interest to register.

**Comment by DethAriel** [comment]
Hello again @Lino-LP ! This bug has now been fixed in `ng-recaptcha@6.1.0`. I wanted to thank you one more time for submitting it, and taking your time to further elaborate on your use-case.

Happy coding!
